### PR TITLE
Feat[CX-2183]: Displaying new Collector Profile info

### PR DESCRIPTION
**Pull request by MrSltun**
The type of this PR is: **Feature**

This PR resolves [CX-2183]

### Description

collector profile

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- display collector profile info - mrsltun

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-2183]: https://artsyproduct.atlassian.net/browse/CX-2183?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 1fc18101ac677e4c29aa9a9a0ab07a33a10933d9
</p>

File: src/palette/svgs/PageIcon.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const PageIcon: React.FC<IconProps> = (props) => {
   return (
     <Icon {...props} viewBox="0 0 18 18">
       <Path
-        d="M7,4.58398438 L3.5,4.58398438 L3.5,1 L1,1 L1,11 L7,11 L7,4.58398437 Z M6.09708454,3.58398437 L4.5,1.94556904 L4.5,3.58398438 L6.09708454,3.58398438 Z M0,0 L4,0 L8,4.10351563 L8,12 L0,12 L0,0 Z"
+        d="M12 7.584H8.5V4H6v10h6V7.584zm-.903-1L9.5 4.946v1.638h1.597zM5 3h4l4 4.104V15H5V3z"
```
Comment (MrSltun) [inline src/palette/svgs/PageIcon.tsx:10]: same comment

File: src/lib/Scenes/MyProfile/MyCollectionAndSavedWorks.tests.tsx [hunk 2]
```diff
@@ -166,5 +166,62 @@ describe("MyCollectionAndSavedWorks", () => {
         expect(wrapper.root.findAllByType(Avatar).length).toEqual(1)
       })
     })
+
+    describe("With Collector Profile feature flag OFF", () => {
```
Comment (MounirDhahri) [inline src/lib/Scenes/MyProfile/MyCollectionAndSavedWorks.tests.tsx:170]: thanks for the thorough tests

File: src/lib/Scenes/MyProfile/MyCollectionAndSavedWorks.tests.tsx [hunk 3]
```diff
@@ -166,5 +166,62 @@ describe("MyCollectionAndSavedWorks", () => {
...
+        })
+
+        const text = extractText(wrapper.root)
+
+        expect(text).toContain("Princess")
+        expect(text).toContain("Member since 2012")
+        expect(text).toContain("Richest Collector! 💰")
+        expect(text).not.toContain("Guardian of the Galaxy")
+        expect(text).not.toContain("Atlantis")
+        expect(text).not.toContain("Marvel Universe")
```
Comment (MounirDhahri) [inline src/lib/Scenes/MyProfile/MyCollectionAndSavedWorks.tests.tsx:196]: **thought(not-a-blocker):** you could have also added a test-id to the wrapper containing those details and looked for it

**Review by MounirDhahri — APPROVED** [review]
LGTM

File: src/palette/svgs/PageIcon.tsx (hunk 1 above)
Comment (olerichter00) [reply src/palette/svgs/PageIcon.tsx:10]: 👍

File: src/lib/Scenes/MyProfile/MyCollectionAndSavedWorks.tsx [hunk 4]
```diff
@@ -116,6 +118,39 @@ export const MyProfileHeader: React.FC<{ me?: MyCollectionAndSavedWorks_me }> =
...
+
+      {showCollectorProfile && (
+        <Flex px={2} mt={2}>
+          <Join separator={<Spacer my={0.5} />}>
+            {!!me?.location?.display && (
+              <Flex flexDirection="row" alignItems="flex-end">
+                <MapPinIcon width={14} height={14} />
+                <Sans size="2" color={color("black100")} px={0.5}>
+                  {me.location.display}
+                </Sans>
```
Comment (olerichter00) [inline src/lib/Scenes/MyProfile/MyCollectionAndSavedWorks.tsx:130]: If I remember correctly we've deprecated `Sans` and try to use `Text` everywhere.

**Review by olerichter00 — DISMISSED** [review]
Looks great 🚀

**Review by olerichter00 — APPROVED** [review]

**Review by MounirDhahri — DISMISSED** [review]

**Pull request merged**